src/widgets/radio.jsx

```diff
@@ -441,7 +440,12 @@ var RadioEditor = React.createClass({
                     <PropCheckBox label="Auto-none of the above"
                                   labelAlignment="right"
                                   noneOfTheAbove={this.props.noneOfTheAbove}
-                                  onChange={this.props.onChange} />
+                                  onChange={(e) => {
+                                    var rand = seededRNG(this.props.problemNum)();
+                                    var index = Math.floor(rand * this.props.choices.length);
```

Comment:
Hi @ariabuckles , thanks for the reply!

I see what you're saying, obviously I'm still missing (a lot of) bits and pieces of how perseus works, apologies. So should I put back these lines in `choiceTransform`? Or do you want to get rid of the random none-of-the-above feature altogether in this PR?

Thanks,
Alessandro